src/actors/build/tf_magnet.jl

```diff
@@ -47,7 +48,8 @@ function tf_required_J_B!(bd::IMAS.build, eq::IMAS.equilibrium)
 
     # current in the TF coils
     current_TF = B0 * R0 * 2π / constants.μ_0 / bd.tf.coils_n
-    TF_cx_area = π * (hfsTF.end_radius^2 - hfsTF.start_radius^2) / bd.tf.coils_n
+    conductor_start_radius = hfsTF.start_radius + bd.tf.nose_thickness*(hfsTF.end_radius-hfsTF.start_radius)
```

Comment:
```suggestion
    conductor_start_radius = hfsTF.start_radius + bd.tf.nose_thickness*hfsTF.thickness
```

Reply:
I don't understand the difference between these two options. Is one prefered?

Reply:
Functionally they are the same, but I rather use `.thickness` if you are referring to the thickness.

`bd.tf.nose_hfs_fraction * hfsTF.thickness` makes people understand right away what you are doing.

`bd.tf.nose_thickness*(hfsTF.end_radius-hfsTF.start_radius)` may be doing the same thing, but it's just harder to quickly grasp..

In general it's better if your code "reads" like what you are doing... if done right, this almost makes comments superfluous.

Reply:
On a more technical level `.thickness` is a value in the tree, while `.end_radius` and `.start_radius` are expressions. This makes `.thickness` is much faster to evaluate.